model/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//model:src/main/proto/preprocess_protos.bzl", "process")
+
+proto_library(
+    name = "proto_lib",
+    srcs = ["src/main/proto/event_logger.proto",
```

Comment:
Nit: newline between ``[`` and the first line.

Reply:
On it! Thanks :)